.gitignore

```diff
@@ -55,3 +55,12 @@ public/sitemap*
 # Ignore local configuration
 .env.development
 .env.test
+
+# Ignore developer cruft
+**/.DS_Store
```

Comment:
OS-specific files should be added to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).